templates/adherent/profile.html.twig

```diff
@@ -82,6 +82,13 @@
         <div class="space--10-0"></div>
 
         <div class="l__row l__row--top l__row--h-stretch l__mobile--col">
+
+            <div class="l__row l__row--h-stretch b__nudge--bottom-tiny form__radio-button">
```

Comment:
why do you need `form__radio-button`? it's not a radio, but select element, no?

Reply:
removed